controller.js

```diff
@@ -24,5 +24,50 @@ module.exports = {
     }).then(countries => {
       res(countries);
     });
+  },
+
+
+/**
+  * creates api log from req body by taking paramters from it
+  * @function setApi
+  * @param {String} userid The userid (primary key) to get a user.
+  * @return {json} status and an object in json format
+  */
+  setApi:function(req,res){
```

Comment:
The function name doesn't seem to reflect the purpose. Shall we change it?

Reply:
Sure,How about addApiLog or addLog? If not both, you can suggest a suitable name.

Reply:
Updated it with the name createApiLog.